test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj

```diff
@@ -39,6 +39,9 @@
 (def master-log-dir
   "./target/master-var/log")
 
+(def multithreaded
+  (= "true" (System/getenv "MULTITHREADED"))
```

Comment:
Missing a paren

Reply:
Oh I think I did this when I updated for Molly's comment, whoops. I'm really hoping it didn't work :P